cmd/mistryd/end_to_end_test.go

```diff
@@ -89,6 +89,48 @@ func waitUntilExists(path string) error {
 	}
 }
 
+func TestBuildRemoveTarget(t *testing.T) {
+	// create files at the target dir
+	target, err := ioutil.TempDir("", "test-remove-target")
+	failIfError(err, t)
+	defer os.RemoveAll(target)
+
+	// create 2 files: a /target/file.txt and a /target/dir/file2.txt
+	dirName := filepath.Join(target, "dir")
+	fileNames := []string{filepath.Join(target, "file.txt"), filepath.Join(dirName, "file2.txt")}
+
+	err = os.Mkdir(dirName, 0755)
+	failIfError(err, t)
+
+	for _, filepath := range fileNames {
+		f, err := os.Create(filepath)
+		failIfError(err, t)
+		f.Close()
+	}
+
+	// run the job with remove-target
+	cmdout, cmderr, err := cliBuildJobTarget(target, "--project", "simple", "--verbose", "--clear-target")
+	t.Logf("cli output: out: %s err: %s", cmdout, cmderr)
+	if err != nil {
+		t.Fatalf("mistry-cli stdout: %s, stderr: %s, err: %#v", cmdout, cmderr, err)
```

Comment:
If I am not mistaken we are using `failIfError` to exit immediately on special cases. You could use the function here, couldn't you?

In addition, if we are going to utilize this helper function we could refactor the tests (in another PR of course). @agis WDYT?

Reply:
I would have to modify `failIfError` if we want to add a custom error message with interpolated variables